main.go

```diff
@@ -159,7 +159,7 @@ func main() {
 	switch config.ChainConfig.OtherChain {
 	case common.ChainEth:
 		if config.EthConfig.SwapType == common.EthSwapTypeEth {
-			otherExecutor = eth.NewEthExecutor(config.EthConfig.Provider, config.EthConfig.SwapContractAddr, config)
+			otherExecutor = eth.NewEthExecutor(config.EthConfig.Provider, config.EthConfig.SwapContractAddr, config.EthConfig)
 		} else {
 			otherExecutor = eth.NewErc20Executor(config.EthConfig.Provider, config.EthConfig.SwapContractAddr, config)
```

Comment:
do we need to use `config.EthConfig` for `Erc20Executor` too

Reply:
No, I didn't change the ERC20Executor becuase it uses a value from the main `ChainConfig` config.
Here is the line where it's used: https://github.com/binance-chain/bep3-deputy/blob/ff5877dc4afc60afdd66eee9a9291057e5c4a3f3/executor/eth/erc20_executor.go#L395